### PR TITLE
Try fixing artifactor NOT logging things properly

### PR DESCRIPTION
**Pull request by mfalesni**
When the formatter change occured, all the log calls were updated to be "conforming to the norm". However, the artifactor logger was optimized for the old way, and it was optimized so much it did not digest the *args at all. This tries to rectify the issue by doing the formatting on the client side. If we ever decide to do something special with the params inside artifactor, this will need to be changed. But I hope it won't be.

### CODE REVIEW

**Comment by psav** [comment]
Can we not collapse the message and "format" it before it goes over the json wire?

**Comment by dajoRH** [comment]
Lint report for commit 22824741e7678954a04d4e9b92687ea2e9a207d6:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav Possibly. I was thinking that you can do something different with the format string on the artifactor side, but this is possible, too.

**Comment by dajoRH** [comment]
Lint report for commit 6d290fc02d40f3785b5024075d5f8d9c130ae231:

`utils/log.py`:
- :red_circle: Line 147:1: **F401** *'safe_string' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav Logs look good, so what I will do is fix the lint, squash it and then it should be ok to go.

**Comment by dajoRH** [comment]
Lint report for commit 41796405181afdccf87fafc710d4c3c57e5d8300:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav @RonnyPfannschmidt I have rebased this one, I am asking for an ack

**Comment by dajoRH** [comment]
Lint report for commit 0bcfd4fa8fe6c3f2e1abce55a4162778598f15ec:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/log.py [hunk 1]
```diff
@@ -470,54 +470,60 @@ def slaveid(self):
         from fixtures.artifactor_plugin import SLAVEID
         return SLAVEID or ""
 
-    def art_log(self, level_name, message, kwargs):
+    def art_log(self, level_name, message, args, kwargs):
+        try:
+            formatted_message = message % args
+        except TypeError:
+            formatted_message = message
+            self.logger.error('Could not format the string %r with %r', message, args)
```
Comment (psav) [inline utils/log.py]: Should this really be an error. If the log message is sent without any args, ie, if it's just a message, that's not an error is it?

Comment (mfalesni) [inline utils/log.py]: Empty args is valid. This is a defensive line, protecting from accidental wrong argument count.

**Pull request merged**